### ISSUE TITLE
Error on MOD + R (rotation shortcut): Could not invoke method (NoSuchMethodException)

### ISSUE THREAD

**Issue by jaysonwcs** [Genymobile/scrcpy#5006]
- [x] I have read the [FAQ](https://github.com/Genymobile/scrcpy/blob/master/FAQ.md).
 - [x] I have searched in existing [issues](https://github.com/Genymobile/scrcpy/issues).

**Environment**
 - OS: Arch Linux
 - scrcpy version: 2.4
 - installation method: Package Manager (pacman)
 - device model: Redmi Note 1o Pro (model M2101K6G / codename sweet)
 - Android version: 14

**Describe the bug**
When I press MOD + R (Alt + R) the screen does not rotate.
It gives an error:
```
scrcpy 2.4 <https://github.com/Genymobile/scrcpy>
INFO: ADB device found:
INFO:     --> (tcpip)  xxxxxxxxxxxxxxx              device  M2101K6G
/usr/share/scrcpy/scrcpy-server: 1 file pushed, 0 skipped. 21.4 MB/s (69007 bytes in 0.003s)
[server] INFO: Device: [Xiaomi] Xiaomi M2101K6G (Android 14)
INFO: Renderer: opengl
INFO: OpenGL version: 4.6 (Compatibility Profile) Mesa 24.1.1-arch1.1
INFO: Trilinear filtering enabled
INFO: Texture: 2400x1080
[server] INFO: Device rotation requested: portrait
[server] ERROR: Could not invoke method
java.lang.NoSuchMethodException: android.view.IWindowManager$Stub$Proxy.freezeRotation [int]
        at java.lang.Class.getMethod(Class.java:2950)
        at java.lang.Class.getMethod(Class.java:2450)
        at com.genymobile.scrcpy.wrappers.WindowManager.getFreezeRotationMethod(WindowManager.java:48)
        at com.genymobile.scrcpy.wrappers.WindowManager.freezeRotation(WindowManager.java:111)
        at com.genymobile.scrcpy.Device.rotateDevice(Device.java:372)
        at com.genymobile.scrcpy.Controller.handleEvent(Controller.java:205)
        at com.genymobile.scrcpy.Controller.control(Controller.java:97)
        at com.genymobile.scrcpy.Controller.lambda$start$0$com-genymobile-scrcpy-Controller(Controller.java:105)
        at com.genymobile.scrcpy.Controller$$ExternalSyntheticLambda1.run(Unknown Source:4)
        at java.lang.Thread.run(Thread.java:1012)
```

**Edited to remove private info.**

**Comment by rom1v** [comment]
Thank you for your report.

It is already fixed in `dev` branch: #4740.

**Comment by jaysonwcs** [comment]
Great!
